db/views/reporting_prescriptions_v03.sql

```diff
@@ -0,0 +1,46 @@
+SELECT p.id as patient_id,
+       p.month_date,
+       COALESCE(max(clean_dosage) FILTER (WHERE clean_name = 'Amlodipine'), 0)          as amlodipine,
+       COALESCE(max(clean_dosage) FILTER (WHERE clean_name = 'Telmisartan'), 0)         as telmisartan,
+       COALESCE(max(clean_dosage) FILTER (WHERE clean_name = 'Losartan Potassium'), 0)  as losartan,
+       COALESCE(max(clean_dosage) FILTER (WHERE clean_name = 'Atenolol'), 0)            as atenolol,
+       COALESCE(max(clean_dosage) FILTER (WHERE clean_name = 'Enalapril'), 0)           as enalapril,
+       COALESCE(max(clean_dosage) FILTER (WHERE clean_name = 'Chlorthalidone'), 0)      as chlorthalidone,
+       COALESCE(max(clean_dosage) FILTER (WHERE clean_name = 'Hydrochlorothiazide'), 0) as hydrochlorothiazide,
+       COALESCE(max(clean_dosage)
```

Comment:
It looks like these changes didn't make it into `structure.sql` -- fixing it in https://github.com/simpledotorg/simple-server/pull/3127.